test/simple.ll

```diff
@@ -126,6 +126,13 @@ entry:
   ret void
 }
 
+; CHECK: "test_array_alloca"
+; Function Attrs: nounwind
+define dso_local void @test_array_alloca() #0 {
```

Comment:
Probably it's worth to outline the test in a separate file with checks for generated SPIR-V and LLVM IR after the translation. Reason: LLVM IR changes after roundtrip (which is fine) and this should be exposed in the test.

Reply:
Created standalone test